src/pages/patientView/timeline2/VAFChart.tsx

```diff
@@ -672,44 +675,72 @@ export default class VAFChart extends React.Component<IVAFChartProps, {}> {
 
     render() {
         return (
-            <svg
-                width={this.props.store.pixelWidth}
-                height={this.recalculateTotalHeight()}
-            >
-                {this.renderData.lineData.map(
-                    (data: IPoint[], index: number) => {
-                        return data.map((d: IPoint, i: number) => {
-                            let x1 = this.xPosition[d.sampleId],
-                                x2;
-                            let y1 = this.yPosition[d.y],
-                                y2;
-
-                            const nextPoint: IPoint = data[i + 1];
-                            if (nextPoint) {
-                                x2 = this.xPosition[nextPoint.sampleId];
-                                y2 = this.yPosition[nextPoint.y];
-                            }
-
-                            let tooltipDatum: {
-                                mutationStatus: MutationStatus;
-                                sampleId: string;
-                                vaf: number;
-                            } = {
-                                mutationStatus: d.mutationStatus,
-                                sampleId: d.sampleId,
-                                vaf: d.y,
-                            };
-
-                            const color = this.groupColor(d.sampleId);
-
-                            return (
-                                <g>
-                                    {x2 && y2 && (
-                                        <VAFPointConnector
-                                            x1={x1}
-                                            y1={y1}
-                                            x2={x2}
-                                            y2={y2}
+            <>
+                {this.props.headerEl && (
```

Comment:
here we take the DOM element represening the header for the track and can render a header to it